app/controllers/licences.controller.js

```diff
@@ -91,9 +92,16 @@ async function viewReturns (request, h) {
   })
 }
 
+async function submitSupplementaryFlag (request, h) {
```

Comment:
We've been reserving `submit` for when a web form is submitted to us rather than for _all_ `POST` requests. So, we can rename this to match the endpoint directly, `supplementaryFlag()`.